### PR TITLE
Final Updates to the TWC 2021 wiki page.

### PR DESCRIPTION
**Pull request by mangomizer**
Hopefully I did this correctly.

### CODE REVIEW

File: wiki/Tournaments/TWC/2021/en.md [hunk 1]
```diff
@@ -31,14 +31,18 @@ The **osu!taiko World Cup 2021** (***TWC 2021***) is a country-based osu!taiko t
 | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $80 per team member, unique profile badge |
 | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $40 per team member, unique profile badge |
 
+![image](https://user-images.githubusercontent.com/30021847/117518638-7b181e00-af98-11eb-8771-32a243ea78f1.png "TWC 2021 winner badge") ![image](https://user-images.githubusercontent.com/30021847/117518656-8ff4b180-af98-11eb-98e5-541283fca1bc.png "TWC 2021 2nd Place badge")
+![image](https://user-images.githubusercontent.com/30021847/117518661-97b45600-af98-11eb-9c4a-285d05fecef6.png "TWC 2021 3rd Place badge")
+
+
```
Comment (ViaSnake) [inline wiki/Tournaments/TWC/2021/en.md:37]: You will probably need to remove this extra line.

**Review by ViaSnake — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/TWC/2021/en.md [hunk 2]
```diff
@@ -31,14 +31,17 @@ The **osu!taiko World Cup 2021** (***TWC 2021***) is a country-based osu!taiko t
 | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $80 per team member, unique profile badge |
 | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $40 per team member, unique profile badge |
 
+![image](https://user-images.githubusercontent.com/30021847/117518638-7b181e00-af98-11eb-8771-32a243ea78f1.png "TWC 2021 winner badge") ![image](https://user-images.githubusercontent.com/30021847/117518656-8ff4b180-af98-11eb-98e5-541283fca1bc.png "TWC 2021 2nd Place badge")
+![image](https://user-images.githubusercontent.com/30021847/117518661-97b45600-af98-11eb-9c4a-285d05fecef6.png "TWC 2021 3rd Place badge")
```
Comment (zeusminus) [inline wiki/Tournaments/TWC/2021/en.md:35]: I don't think this is an appropriate way of inserting images. You need to upload them to a folder named `img` under `TWC/2021` and then replace these URLs with `/wiki/Tournaments/TWC/2021/img/<filename>.png`

File: wiki/Tournaments/TWC/2021/en.md [hunk 3]
```diff
@@ -88,20 +91,7 @@ The osu!taiko World Cup 2021 is run by various community members.
...
-
-| Team A | Team B | Match time | Local time A | Local time B |  |
-| --: | :-- | :-: | :-: | :-: | :-: |
-| Brazil ![][flag_BR] | ![][flag_KR] South Korea | **01:00 UTC** | 22:00 (Sat.) UTC-03 | 10:00 UTC+09 | ¹ |
-| Japan ![][flag_JP] | ![][flag_KR] South Korea | **06:00 UTC** | 15:00 UTC+09 | 15:00 UTC+09 | ² |
-| Japan ![][flag_JP] | ![][flag_BR] Brazil | **15:00 UTC** | 00:00 (Mon.) UTC+09 | 12:00 UTC-03 | ² |
-
-¹ 3rd place encounter\
-² Encounter dependent on results of ¹3rd place encounter
+![TWC 2021 Podium](https://user-images.githubusercontent.com/30021847/117518465-bc5bfe00-af97-11eb-886b-bdee59a4f91b.png)
```
Comment (zeusminus) [inline wiki/Tournaments/TWC/2021/en.md:94]: Same as above.

**Review by zeusminus — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/TWC/2021/en.md (hunk 3 above)
Comment (mangomizer) [reply wiki/Tournaments/TWC/2021/en.md]: fixed

File: wiki/Tournaments/TWC/2021/en.md (hunk 2 above)
Comment (mangomizer) [reply wiki/Tournaments/TWC/2021/en.md]: fixed

**Comment by peppy** [comment]
The podium image is still 800+kb. Rather than attempting to compress as png, it should be converted to a high quality jpeg (in line with every previous tournament).

**Review by peppy — APPROVED** [review]

**Pull request merged**